toqito/channels/generalized_amplitude_damping.py

```diff
@@ -0,0 +1,83 @@
+"""Generates the (generalized) amplitude damping channel."""
+
+import numpy as np
+
+
+def generalized_amplitude_damping(

```

Comment:
I would prefer to call this `amplitude_damping` instead of `generalized_amplitude_damping`. This would apply to the file names as well.